pages/base-page.js

```diff
@@ -37,6 +37,7 @@ exports.BasePage = class BasePage {
     ).first();
     this.addColumnGridLayoutBtn = page.locator('g[class*="grid-plus-button"]',
     ).last();
+    this.columnsRowsOnDesignPanel = page.getByTestId('inspect-layout-rows').locator('..');
```

Comment:
This locator is needed to capture a screenshot of a fragment of the design panel. I couldn’t find another way to capture this fragment since it has no unique name or ID, although its child element does. Using locator('..') helps to locate the parent element.

Reply:
Ok!